### PR TITLE
updated browser/clientside

### PR DESCRIPTION
**Pull request by aksiere**

### CODE REVIEW

File: client_packages/cef/login/index.html
Comment (MrPancakers) [inline client_packages/cef/login/index.html]: I see you've replaced the local files for CDN links, is there a reason why you choose to use external resources? Using external files can add additional loading time onto users which is why CDN's are avoided in this resource.

Also, we already use bootstrap v4.5 and this is v4.

As of now, I wouldn't merge this with CDN links, I'd want these to be local files (see the cef/assets folder)

**Review by MrPancakers — CHANGES_REQUESTED** [review]
CDN links aren't necessary here, if you want to update the jQuery resource you can. Otherwise, revert those changes and I can test it out and merge if all works out.

**Review by MrPancakers — CHANGES_REQUESTED** [review]
Not sure why but after all your changes the pages don't even work in-game anymore. I cannot submit anything so this needs to be looked into. Try testing in-game before submitting changes, I can try debug later but for now I still cannot merge this.

**Comment by aksiere** [comment]
refactored.
it works for me

**Review by MrPancakers — APPROVED** [review]

**Comment by MrPancakers** [comment]
> refactored.
> it works for me

Awesome tested again and it's all working, just need to fix up the error messages as they aren't user friendly but once I've done that I'll merge it. Thanks

Just an additional note and not a big deal but if possible try not to do one commit per file, if all the file changes are related put them under one commit. You can contact me on Discord if you're unsure how to do that.

**Pull request merged**
